bedrock/products/templates/products/vpn/platforms/desktop.html

```diff
@@ -41,124 +41,113 @@
 
 {% block content %}
 <main>
-  <section class="vpn-hero">
-    <div class="mzp-l-content mzp-t-content-xl">
-      <div class="vpn-hero-container">
-        <div class="vpn-hero-body">
-          <h1 class="vpn-hero-title">{{ ftl('vpn-platforms-shared-title') }}</h1>
-          <h2 class="vpn-hero-subtitle">{{ ftl('vpn-desktop-hero-headline') }}</h2>
-          <div class="vpn-hero-desc">
-            <p>{{ ftl('vpn-desktop-page-description') }}</p>
-
-            <a class="vpn-button" href="{{ url('products.vpn.landing') }}">{{ ftl('vpn-platforms-shared-cta') }}</a>
-
-            <div class="vpn-hero-guarantee">
-                <em>{{ ftl('vpn-platforms-shared-day-guarantee') }}</em>
-            </div>
-          </div>
-        </div>
-        <div class="vpn-hero-image hide-from-legacy-ie" data-illustration="n-1">
-          <picture>
-            <source srcset="{{ static('img/products/vpn/platforms/vpn-hero-desktop-high-res.png') }} 480w, {{ static('img/products/vpn/platforms/vpn-hero-desktop-high-res.png') }} 960w" media="(min-width: 1024px)">
-            <source srcset="{{ static('img/products/vpn/platforms/vpn-hero-desktop.png') }}" media="(max-width: 1023px)">
-            <img src="{{ static('img/products/vpn/platforms/vpn-hero-desktop.png') }}" class="n-1" alt="" width="480" height="480">
-          </picture>
-        </div>
-      </div>
-    </div>
-  </section>
+  {% block hero %}
+    {% call vpn_platform_hero(
+      heading=ftl('vpn-shared-product-name'),
+      subheading=ftl('vpn-desktop-hero-headline'),
+      desc=ftl('vpn-desktop-page-description'),
+      image_url='img/products/vpn/platforms/vpn-hero-desktop.png'
+    ) %}
+      <a class="mzp-c-button mzp-t-xl" href="{{ url('products.vpn.landing') }}">{{ ftl('vpn-shared-platform-cta-button') }}</a>
+      <p class="guarantee-copy js-text-vpn-fixed-price-available">
```

Comment:
Try not to rely on conditional classes such as `js-text-vpn-fixed-price-available`. Those are undergoing quite a big refactor to support the new pricing models, so I would try and keep code related to this logic out of these pages for now.

Reply:
`js-text-vpn-fixed-price-available` was copy/paste artifacting. I'll remove it.

Reply:
Removed!